cupy/lib/_routines_poly.py

```diff
@@ -104,6 +104,16 @@ def polymul(a1, a2):
     return cupy.convolve(a1, a2)
 
 
+def _polypow(x, n):
+    if n == 0:
+        return 1
+    if n == 1:
+        return x
+    if n % 2 == 0:
+        return _polypow(cupy.convolve(x, x), n / 2)
```

Comment:
Please use integer division.
```suggestion
        return _polypow(cupy.convolve(x, x), n // 2)
```